zlib-1.2.8/CMakeLists.txt

```diff
@@ -204,3 +204,16 @@ elseif(BUILD_SHARED_LIBS AND WIN32)
     # Creates zlib1.dll when building shared library version
     set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
 endif()
+
+add_library(ZLIB::header INTERFACE IMPORTED GLOBAL)
+if(CMAKE_GENERATOR MATCHES "Xcode")
```

Comment:
when I replaced `CMAKE_CURRENT_SOURCE_DIR` with `CMAKE_CURRENT_BINARY_DIR` that broke zlib build on windows, since the `CMAKE_CURRENT_BINARY_DIR` contained `zconf.h` but that directory wasnt included correct in `Play!` as it was setup in `framework`.
this was an attempt at including the headers in a library to make it easier to include in the project irrespective of `zconf.h`

I still didnt get the chance to reboot to osx to see how this code worked, although zlib wasnt broken for some reason by the `CMAKE_CURRENT_BINARY_DIR` change. ill have a look at this in a few.